util/ipgen/README.md

```diff
@@ -88,13 +91,13 @@ CAPI=2:
 name: ${instance_vlnv("lowrisc:ip:rv_plic")}
 ```
 
-After processing, the `name` key is set to e.g. `lowrisc:opentitan:top_earlgrey_rv_plic`.
+After processing, the VLNV could become `lowrisc:opentitan:top_earlgrey_rv_plic`.
```

Comment:
Nit: I think we probably want "would become" (since it's definitely what would happen)?

Reply:
It is conditional because this will only happen if the top is earlgrey.